Sources/Identity/Identifier/Identifier+Macros.swift

```diff
@@ -14,9 +14,9 @@ import Foundation
 ///  - Backing: The type that backs the ``Identifier`` type, its `RawValue`.
 ///  - name: The name of the new ``Identifier`` type. Must be a valid type identifier at compile time.
 @freestanding(declaration, names: arbitrary)
-public macro Identifier<Backing: Hashable & Codable & Sendable>(_ name: StaticString) = #externalMacro(
+public macro ID<Backing: Hashable & Codable & Sendable>(_ name: StaticString) = #externalMacro(
```

Comment:
Given the issue with collisions, i'd be a little concerned with either of these still colliding with other libraries in the future.

Reply:
Yeah, I do want to keep it straightforward though, and I'm unclear about how much use `#ID` is going to get anyway since it can't really capture i.e. access controls.